std/experimental/ndslice/selection.d

```diff
@@ -707,6 +708,58 @@ pure nothrow unittest
 }
 
 /++
+Returns an m-dimensional slice composed of n-dimensional identical slices,
+where `m` is a number of dimension passed to the `clones`.
+`clones` can be generalized with other selectors.
+
+Params:
+    lengths = dimensions of clones
+Returns:
+    packed `M`-dimensional slice composed of `N`-dimensional slices
```

Comment:
Small nitpicks
1) "grammar: `m` is the ~~a~~ number of dimension__s__"
2)  I think it would helper readers if `m` occurs in `Params`, e.g. "length = m dimensions of clones"
3) You use both lower and uppercase forms of `m` - on purpose?